cmd/config.go

```diff
@@ -303,6 +303,47 @@ func getServiceConfig(reader *bufio.Reader) (*v3.RKEConfigServices, error) {
 		return nil, err
 	}
 	servicesConfig.Kubelet.InfraContainerImage = infraPodImage
+
+	// Add the flexvolume mount to kublet ExtraBinds
+	flexVolMount, err := getConfig(reader, "Enable the FlexVolume driver mount", "no")
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.ContainsAny(flexVolMount, "Yes y Y yes") {
```

Comment:
@dahendel seems like we can create a constant for `Yes y Y yes` as its use repeated